conanfile.py

```
@@ -44,3 +53,12 @@ def requirements(self):
 
         if self.options.with_cmake:
             self.tool_requires("cmake/3.29.3")
+
+    def generate(self):
+        deps = CMakeDeps(self)
```

Ruff (0.12.2)

Undefined name `CMakeDeps`

(F821)